packages/js/tracks/src/index.ts

```diff
@@ -31,20 +36,24 @@ export function recordEvent(
 	tracksDebug( 'recordevent %s %o', 'wcadmin_' + eventName, eventProperties, {
```

Comment:
I wonder if we should change the location for this as well, or add an extra attribute `isValid` or something like that.
This is not a must for this PR, but I did find it confusing to still see the invalid tracks logged.
In development mode it's fine as it shows the errors after, in production it doesn't show the errors so it's more confusing.